axelrod/strategies/__init__.py

```diff
@@ -10,14 +10,19 @@
     MetaMajorityMemoryOne, MetaWinnerMemoryOne, MetaMajorityFiniteMemory,
     MetaWinnerFiniteMemory, MetaMajorityLongMemory, MetaWinnerLongMemory,
     MetaMixer
-    )
+)
 
-all_strategies.extend([MetaHunter, MetaMajority, MetaMinority, MetaWinner,
-                       MetaMajorityMemoryOne, MetaWinnerMemoryOne,
-                       MetaMajorityFiniteMemory, MetaWinnerFiniteMemory,
-                       MetaMajorityLongMemory, MetaWinnerLongMemory, MetaMixer])
+all_strategies.extend([
+    MetaHunter, MetaMajority, MetaMinority, MetaWinner, MetaMajorityMemoryOne,
+    MetaWinnerMemoryOne, MetaMajorityFiniteMemory, MetaWinnerFiniteMemory,
+    MetaMajorityLongMemory, MetaWinnerLongMemory, MetaMixer])
 
 
+def strategy_id(strategy):
+    return strategy.name.lower().replace(' ', '_')
+
+strategy_index = {strategy_id(s): s for s in all_strategies}
```

Comment:
It makes me think of an index at the back of a book, or a database index - something where you look things up!!

Not too bothered myself, if there's a better suggestion.